web/src/engine/src/js-processor/jsKeyboardInterface.ts

```diff
@@ -808,15 +808,17 @@ export class JSKeyboardInterface extends KeyboardHarness {
     this.output(dn, textStore, '');
   }
 
+  // TODO-web-core: Why is this function on the keyboard interface and not the
```

Comment:
Because it's the JS-keyboard API function `KO`, used for the "output" portion of JS keyboard rule implementations for Keyman keyboards.

Reply:
Refer to https://github.com/keymanapp/keyman/blob/9c9696c5fdcf2e90c7c87ffa3bfc744263bf82c2/web/src/test/manual/web/web_context_tests.js#L1457-L1463 for an example from a (debug-compiled) JS keyboard.